src/services/storage/drivers/relational/index.ts

```diff
@@ -62,25 +64,42 @@ export class RelationalStorage implements StorageDriver {
 	}
 	url = new (class RelationalUrlStorage {
 		constructor(public storage: RelationalStorage) {}
-		public async get(id: string): Promise<StoredUrl> {
-			const storedUrl = await this.storage.db
-				.table<StoredUrl & { serial?: number }>('urls')
-				.select('*')
-				.where('id', id)
-				.first()
 
-			delete storedUrl?.serial
+		public async get(id: string, options = { withInfo: false }): Promise<StoredUrl | UrlWithInformation> {
+			let storedUrl, urlInfo
+			if (!options.withInfo) {
+				storedUrl = await this.storage.db
+					.table<StoredUrl & { serial?: number }>('urls')
+					.select('*')
+					.where('id', id)
+					.first()
+
+				delete storedUrl?.serial
+				if (!storedUrl) throw NotFoundError()
```

Comment:
This `if` should probably be down, right before we return.

Reply:
done